voting_ml/data.py

```diff
@@ -58,16 +87,46 @@ def all_data(self, question_list=None):
 
             # get the data from the table and add it to the result
             if question in self._cols_with_string_answers:
-                return_table[:, i] = self._answers_as_ints(answers, question)
+                if(self._convert_to_int):
+                    return_table[:, i] = self._answers_as_ints(answers, question)
+                else:
+                    return_table[:, i] = answers
+            elif(question=='ppage'):
+                return_table[:, i] = self.categorize_age(np.array(answers))
             else:
                 return_table[:, i] = answers
 
         # remove all rows with a nan value in them
         if self._remove_nan:
+            #if(self._convert_to_int):
```

Comment:
Because in the feature selection, we do ordinal encoding, we might not need to convert to integers. But if we want to convert it before, it should be okay. That's why I had removed it for now.